import_export/results.py

```diff
@@ -62,6 +62,49 @@ def __init__(self):
         #: This value is only set for updates.
         self.original = None
 
+    def is_update(self):
```

Comment:
This bunch of copy & paste code makes me a little uncomfortable as above. 

Is it really necessary? Could the code above just check the .action_type itself?

Reply:
I added these helper methods to avoid the proliferation of checks such as: 

```
if row_result.import_type == RowResult.IMPORT_TYPE_UPDATE:
  ...
```

which you see in previous [`admin.py`](https://github.com/django-import-export/django-import-export/blob/0dc824abc526037e330813e901df27dd35bd14ac/import_export/admin.py#L230).  Although it looks like a fair bit of duplication, I think it is cleaner interface because it hides implementation of `row_result`.  However it is a minor point.